avalanche-ops/src/aws/cfn-templates/vpc.yaml

```diff
@@ -56,6 +56,11 @@ Parameters:
     Type: Number
     Default: 9650
     Description: HTTP port
+  
+  UserProvidedPort:
```

Comment:
This is not being used below? We need to create the security group ingress rule, as we do for SSH.